### DISCUSSION TITLE
HTTPS / LetsEncrypt Question

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#7482] [Q&A] [answered]
I'm not a website guy so this might be a stupid question, but if I have what seems to be an up to date certificate from the LetsEncrypt section, then why is my Azuracast installation site still HTTP? I am seeing the same thing on my website host, does this mean I need to get an SSL for my Azuracast domain through them too? My Azuracast installation is on a separate server from my web host, do I need an SSL certificate stored on my Azuracast server (separately from whatever Lets Encrypt certs get pulled)?

The problem- My embedded player and history were both showing expired certificate errors (All my radio web pages have SSL certificates through my host, except my Azuracast domain). I went into Azuracast and changed the base URL from either HTTP to HTTPS or viceversa (can't remember nor replicate the issue), and I switched "Use Web Proxy for Radio" on, and "Always Use HTTPS" on. The players now work fine on PC, and I can't replicate the issue no matter what settings I switch back. But on my mobile pages it is just showing blank in the player and history spots. Trying to figure out exactly where the issue might be, so any thoughts you might have on this would be very helpful. Thank you!

**Comment by Heimatradio** [comment, marked as answer]
1. in Administration - System Settings - Services you have to set at domain name(s) your domain e.g. : your_azuracast_domain.com
2. save the changes below
3. click on the butten GENERATE/RENEW CERTIFICATE
Normally https will work, in some cases a server restart helps
Insert all links to Azuracast (player, etc.) with https on your website
The SSL certificate on the Azuracast server is sufficient!!!!!!!!!!!!!!!

**Reply by ghost** [reply]
Thanks for the reply- I did all that, but good to know the SSL cert on the Azuracast server is good enough. My Azuracast is now saying it is verified by Lets Encrypt, so all good there, I think it just took a full on server reboot. 

All was well, but then last night my song titles stopped updating for about 2 hours, and then when I reconnected BuTT encoder to the stream, it failed. Replacing the domain name with my Azura IP address (in BuTT) fixed the issue, so... DNS problem? On PC this works fine, but on my mobile pages- which have the exact same embed data and have worked before- says the server site can't be found. Probably more of a DNS issue than an SSL issue, but... any thoughts on that from the Azuracast side?

**Reply by Heimatradio** [reply]
I think it's a server problem. Is the server running without CPU or RAM overload? 

![Unbenannt-2](https://github.com/user-attachments/assets/2b2f2ffd-8c6f-4482-accc-abaa5d886aee)
looks that good at your server?

What is the configuration (CPUs and RAM)?

**Reply by ghost** [reply]
Good idea! But the server status looks good, overall CPU usage around 1%-5%, I have 4 CPUs and they are all around 0%-1%. I see occasional spikes up to around 15% total usage, bumping some of my CPUs to around 30% but that only lasts a second. This server has 4GB of RAM, hosted through Cloudfanatic. Please also see my updated post #7488 for more details. Thanks!
